Shoko.Server/Models/SVR_VideoLocal.cs

```diff
@@ -235,7 +235,7 @@ public void SetResumePosition(long resumeposition, int userID)
 
         public void ToggleWatchedStatus(bool watched, int userID)
         {
-            ToggleWatchedStatus(watched, true, null, true, userID, true, true);
+            ToggleWatchedStatus(watched, true, watched ? DateTime.Now : null, true, userID, true, true);
```

Comment:
Added the date when it's watched so both the anidb file, the file and the episode will have the exact same date.